.gitignore

```diff
@@ -15,6 +15,8 @@
 *.exe
 build/
 tracy_build
+source/agora/cli/version/version_build
+source/scpp/cpp_build
```

Comment:
These files were created when I run `dub build`..

Reply:
Yes they are binaries. What's interesting is that they were not created before, or we would have seen it. So maybe verbose mode changed something ?

Reply:
Might as well try without the verbose mode.